### PR TITLE
feat(network): `static_routes` field to router

### PR DESCRIPTION
**Pull request by villevsv-upcloud**

### CODE REVIEW

File: upcloud/network.go [hunk 1]
```diff
@@ -292,6 +292,14 @@ type Router struct {
...
 	UUID             string             `json:"uuid"`
 	Labels           []Label            `json:"labels"`
+	StaticRoutes     []StaticRoute      `json:"static_routes"`
+}
+
+// StaticRoute represents a Static route
+type StaticRoute struct {
+	Name    string `json:"name"`
+	Route   string `json:"route"`
+	Nexthop string `json:"nexthop"`
```
Comment (peknur) [inline upcloud/network.go:302]: Should we use NextHop ? :thinking:

**Review by peknur — APPROVED** [review]
Looks good, left small open question but it's not a blocker :ok_hand:

**Review by ajmyyra — APPROVED** [review]
Looking nice! Only issue I noticed was already commented by Pekka and I do agree that it's not really a blocker.

File: upcloud/network.go (hunk 1 above)
Comment (villevsv-upcloud) [reply upcloud/network.go:302]: I opted to use `Nexthop` for the field (`nexthop`) conversion, as it the original does not contain underscore. I do agree that `NextHop` would "look" better, but I would still keep it as it is to follow previous conventions.

**Pull request merged**